src/sbl_filing_api/services/multithread_handler.py

```diff
@@ -25,7 +25,12 @@ async def check_future(future, submission_id, exec_check):
         future.cancel()
         exec_check["continue"] = False
         await repo.error_out_submission(submission_id)
-        logger.error(f"Validation for submission {submission_id} did not complete due to an unexpected error.")
+        logger.error(
+            f"Validation for submission {submission_id} did not complete due to an unexpected error.",
+            future.exception(),
+            exc_info=True,
+            stack_info=True,
+        )
```

Comment:
after some digging, looks like we might have been doing exception logging wrong in quite a few places; this probably still shows up in the logs because we'd probably encounter `During handling of the above exception, another exception occurred:` message which then still prints out the trace; gonna do some more digging before I create tickets for the API repos, but can you check if that message shows up about `not all arguments converted during string formatting`?
apparently the subsequent args after the string, in this case where `future.exception()` is, is meant to format the message. i.e.
```python
log.error("test message: %s", some_stringable_variable)
```
 `log.exception` / `log.error(exc_info=True, stack_info=True,)` would ideally be encased within the `except Exception` block, and it automatically prints out the error it was encased in; so we don't have to pass the exception into the logging method; and we'd just need to add the custom message.
For this to work with futures, this seems to be the correct way given what I've mentioned above:
```python
try:
    future.result()
except Exception:
    logger.exception("some custom message")
```